public/canjs/controls/select-issue-type/select-issue-type.js

```diff
@@ -90,111 +90,30 @@ export class SelectIssueType extends StacheElement {
             </button>
         {{/ }}
     `;
-    static props ={
-        simplifiedIssueHierarchy: {
-            async(){
-                return getSimplifiedIssueHierarchy({
-                    isLoggedIn: this.jiraHelpers.hasValidAccessToken(),
-                    jiraHelpers: this.jiraHelpers,
-                });
-            }
-
+    static props ={     
+        routeData: {
+          get default() { return routeData; }
         },
-        
         get issueHierarchy(){
             return this.derivedIssues && this.derivedIssues.length ?
```

Comment:
same with derived issues, I doubt that needs to be passed around and can just be pulled from routeData

Reply:
I'm going to add a subtask to make sure we are not passing stuff around that is already in routeData.